SpatialGDK/Documentation/content/non-unreal-server-worker-types.md

```diff
@@ -106,7 +115,7 @@ You could serialize and send a command response in your Unreal project code in t
 
 
 ```
-Worker_RequestId SendSomeCommandRequest(Worker_EntityId TargetEntityId, Worker_ComponentId ComponentId, Schema_FieldId CommandId) {
+Worker_RequestId SendSomeCommandResponse(Worker_EntityId TargetEntityId, Worker_ComponentId ComponentId, Schema_FieldId CommandId) {
```

Comment:
In this example could you perhaps show something cleaner using `AddStringToSchema`. Not a big deal, but it will probably help someone trying to write such code discover the `SchemaUtils` and see more examples of how to work with schema objects.

Reply:
Hmm, schema utils looks good but it's a pretty limited subset of functionality that I'd be scared to see change and accidentally break the docs. I'd prefer to play it safer for now and just use the direct C API calls. Could reference the SchemaUtils file in the doc?